### PR TITLE
feat(Guild): support addGuildMember

### PR DESCRIPTION
**Pull request by ebroder**
Not sure how much explanation is required here, but I'm looking to switch an application of mine to use Eris and this is an endpoint that I'm using today.

### CODE REVIEW

**Comment by coolcalcacol** [comment]
Lol.

**Comment by jaw0r3k** [comment]
You forgot to add typings for `addUser`

**Comment by ebroder** [comment]
> You forgot to add typings for `addUser`

Good flag, thanks. Should be fixed now.

File: index.d.ts [hunk 1]
```diff
@@ -2802,6 +2809,7 @@ declare namespace Eris {
     widgetEnabled?: boolean | null;
     constructor(data: BaseData, client: Client);
     addDiscoverySubcategory(categoryID: string, reason?: string): Promise<DiscoverySubcategoryResponse>;
+    addMember(userID: string, accessToken: string, options?: AddGuildMemberOptions): Promise<void>;
```
Comment (jaw0r3k) [inline index.d.ts:2819]:
```suggestion
    addUser(userID: string, accessToken: string, options?: AddGuildMemberOptions): Promise<void>;
```

**Comment by abalabahaha** [comment]
`addMember` would be more consistent with `addGuildMember`, did I miss something?

**Comment by ebroder** [comment]
Hmm. Yeah sure, that seems reasonable. I think in my head I was naming the method after the noun that it acts on (since you pass in a _user_ ID), but yeah I agree that parity makes more sense. Pushed a fix (and a catchup to `dev`)

**Review by abalabahaha — APPROVED** [review]

**Comment by abalabahaha** [comment]
Yeah member/user is a bit more iffy here. Though I think more of the lib has endpoint-related names, so let's go with member for now.

**Pull request merged**